qgis 3.14.15 build failure: "Policy "CMP0071" is not known to this version of CMake"

**Describe the bug**
The main CMakeLists.txt file claims to require cmake "3.1.0" as minimum, but then it sets "CMP0071": this policy was introduced in cmake 3.10 and is not recognized.
This situation causes a build failure when using any cmake < 3.10 (in my building setup cmake 3.7.2)... 
Perhaps a typo ("3.1.0" instead of "3.10)"? Or someone forgot to update the cmake requirement?

**How to Reproduce**
1. Install the sources
2. Run cmake (< 3.1.0) with the desired options
3. Watch the error: "CMake Error at CMakeLists.txt:11 (CMAKE_POLICY):
                              Policy "CMP0071" is not known to this version of CMake."

**QGIS and OS versions**
QGIS 3.14.15
Rosa Linux Fresh R11 [a Mandriva derivative] x86_64

**Additional context**
Trying to build latest QGIS sources to update distribution package.
The issue is still present in the master branch.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 #############################################################
 # CMake settings
-CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.10.0)
 SET(CMAKE_COLOR_MAKEFILE ON)
 SET(CMAKE_AUTORCC ON)
 # set path to additional CMake modules

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -48,7 +48,7 @@ Following a summary of the required dependencies for building:
 
 Required build tools:
 
-* CMake >= 3.0.0
+* CMake >= 3.10.0
 * Flex >= 2.5.6
 * Bison >= 2.4
 * Python 3

